src/v2/components/Account/Detail/index.jsx

```diff
@@ -0,0 +1,120 @@
+// @flow
+import {observer} from 'mobx-react-lite';
+import {Container, IconButton, Tabs, useTheme} from '@material-ui/core';
+import useMediaQuery from '@material-ui/core/useMediaQuery/useMediaQuery';
+import {map, eq} from 'lodash/fp';
+import React, {useState} from 'react';
+import {Match} from 'react-router-dom';
+import {ReactComponent as StarIcon} from 'v2/assets/icons/star.svg';
+import SectionHeader from 'v2/components/UI/SectionHeader';
+import HelpLink from 'v2/components/HelpLink';
+import QRPopup from 'v2/components/QRPopup';
+import CopyBtn from 'v2/components/UI/CopyBtn';
+import Loader from 'v2/components/UI/Loader';
+import ApplicationDetailStore from 'v2/stores/applications/detail';
+import {LAMPORT_SOL_RATIO} from 'v2/constants';
+import TabNav from 'v2/components/UI/TabNav';
+import Chart from './Chart';
+
+import Transactions from './Transactions';
+import AccountCode from './Code';
+import useStyles from './styles';
+
+const AccountDetail = ({match}: {match: Match}) => {
+  const classes = useStyles();
+  const {
+    isLoading,
+    applicationId,
+    accountInfo,
+    applicationView,
+  } = ApplicationDetailStore;
+
+  if (applicationId !== match.params.id) {
+    ApplicationDetailStore.init({applicationId: match.params.id});
+  }
+
+  const [tab, setTab] = useState(0);
+  const theme = useTheme();
+  const verticalTable = useMediaQuery(theme.breakpoints.down('xs'));
+
+  if (isLoading) {
+    return <Loader width="533" height="290" />;
+  }
+
+  const handleTabChange = (event, tab) => setTab(tab);
+
+  const specs = [
+    {
+      label: 'Balance',
+      hint: '',
+      value: `${(accountInfo.lamports * LAMPORT_SOL_RATIO).toFixed(8)} SOL`,
+    },
+    {
+      label: 'Transactions',
+      hint: '',
+      value: `${(accountInfo.lamports * LAMPORT_SOL_RATIO).toFixed(8)} SOL`,
```

Comment:
thinking maybe is this supposed to be some other data point? 😕

Reply:
marked as to do for now